docs/UserGuide.md

```diff
@@ -204,7 +204,9 @@ Examples:
 
 ### Undo previous entry : `undo`
 
-Undo the previous command, stackable. (Able to keep undo-ing till there are no commands left to be undone)
+Undo the previous command, stackable (all commands that were successfully executed previously can be continuously undone until there are no commands left to be undone).
+The undo/redo command is affected by all successful commands, whether they change the data or not.
```

Comment:
I think we should only specify this behaviour for the undo command in this section, since we have a separate section to describe the redo behaviour in the UG